bootcamp/tutorials/integration/function_calling.ipynb

```diff
@@ -0,0 +1,430 @@
+{
```

Comment:
<p>The story of doing chats over flights, and the schema definition of functions are not trivial. Some illustration is necessary, e.g. explain what the functions do and the idea of defining the "API" of function through natural language, which is kind of the core idea behind this blog.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>